msgeq7.cpp

```diff
@@ -30,8 +30,8 @@ float scaleValue(float value, float value_min, float value_max) {
   value_max = (value_max - value_min);
   value = (value / value_max);
 
-  value = min(1.0, value);
-  value = max(0.0, value);
+  value = min(1.0, (double) value);
```

Comment:
Would be more efficient (esp32) to change the expression to float. You just need to append "f" to literal values - so "1.0" --> "1.0f".

`value = min(1.0f, value);`